<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>400</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(213, 244, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QPushButton" name="button_chu">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px ;background-color:gray;
</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="button_cheng">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:gray;</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_jian">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:gray;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_jia">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:gray;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="button_2">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="button_6">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="button_1">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(229,229,229);border-radius:12px;
hover{
	background-color:pink;
	color:white;
};
</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="button_0">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>103</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;
background-color:rgb(229,229,229);
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="4">
       <widget class="QPushButton" name="button_dengyu">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>8000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(232,111,30);</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="button_dian">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>·</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="button_3">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="button_9">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button_7">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="button_8">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="button_5">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="button_4">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:12px;background-color:rgb(229,229,229);</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>35</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border:4px solid black;border-radius:12px;
</string>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
